addons/crm/models/crm_lead.py

```diff
@@ -66,7 +66,7 @@ def _default_stage_id(self):
     name = fields.Char('Opportunity', required=True, index=True)
     partner_id = fields.Many2one('res.partner', string='Customer', track_visibility='onchange', track_sequence=1, index=True,
         help="Linked partner (optional). Usually created when converting the lead. You can find a partner by its Name, TIN, Email or Internal Reference.")
-    active = fields.Boolean('Active', default=True, track_visibility=True)
+    active = fields.Boolean('Active', default=True)
```

Comment:
Removing the tracking you break the opportunity lost subtype mechanism and tracking. If you look at the code that triggers the opportunity lost subtype
```
'active' in init_values and self.probability == 0 and not self.active
```
If you don't track active you break this. Is it what is requested ? Either we remove the opportunity lost subtype, either we have to keep tracking active.

Reply:
Ok so we let it go